backend/src/models/reports/genericModelTemplate.js

```diff
@@ -6,63 +6,81 @@
  * you can adapt this to help structure your model.
  * this is based on Tab_20_rpt_PA_Billed.js AND Tab_50_rpt_PF_NetRecoverySummaryByQuarter.js:w
  *
+ * This template tries to just use the raw query, but look at tab 20 or tab 21 if you want
+ * to wrap it further in knex() methods.
+ *
  */
 // libs
 const { knex } = require("@database/databaseConnection")();
 
-/**
- * Base query to re-use for report & report totals
- *
- * @param   {number | string | Array} parameter from the frontend, e.g.: Date, Portfolio(s), fiscal_year, or the like
- * @returns {Knex.QueryBuilder}                 Knex query builder for fetching report totals.
- */
-/*
-const baseQuery = knex("jv")
-  .select({
-    fy: "fiscal_year_id",
-    fiscal_year: "fiscal_year.fiscal_year",
-    project_number: "P.project_number",
-    project_name: "P.project_name",
-  })
-  .leftJoin("project AS P", "P.id", "jv.project_id")
-  .groupBy("fiscal_year_id", "fiscal_year.fiscal_year", "P.project_number", "P.project_name")
-  .orderBy("P.project_number", "asc")
-  .orderBy("fiscal_year", "desc");
- */
-
 /**
  * Retrieves the data for various financial metrics based on the fiscal year.
  *
  * Uses baseQuery twice, for DRYness
  *
- * @param   {number | string | Array} Parameter- The fiscal, Date, Portfolio(s) to grab data for
+ * @param   {number | string | Array} Parameter- The fiscal, Date, or Portfolio(s) to grab data for
  * @returns {Promise}                            - A promise that resolves to the query result
  */
 const reportQueries = {
   fiscalYear: (PARAMETER) =>
     knex("fiscal_year").select("fiscal_year").where("fiscal_year.id", PARAMETER).first(),
-  /*
+
   report: (PARAMETER) =>
-    knex.select("something").from(baseQuery.as("base")).where("someColumn", PARAMETER),
+    knex
+      .select()
+      .fromRaw(
+        `
+     (
```

Comment:
Formatting looks a little weird here, but I don't think it matters as it seems to be just for example anyway

Reply:
yeah the auto-formatting gets wierd when i'm using a string literal. I can clean it up